.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,68 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [main, develop]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [main, develop]
+  schedule:
+    - cron: "19 23 * * 5"
```

Comment:
Crontab guru tells me this is "run every Friday at 23:19" any reason why? We can remove it otherwise

Reply:
I just saw that cas-registration was running it daily, but yeah I think it would be fine just on push and PR